map_resolution: 0.05
world_frame: "odom"
pointcloud_topic_queue_length: 10
pointcloud_queue_max_wait_for_transform_s: 1.0
map_visualization_period_s: 10.0
map_evaluation_period_s: 10.0
map_ground_truth_path: "../catkin_ws/data/wavemap_2d_ground_truth_1cm_floodfill"
